lib/gui/app.js

```diff
@@ -200,9 +200,21 @@ app.run(($timeout) => {
     // otherwise we risk presenting the same error over
     // and over again to the user, while also heavily
     // spamming our error reporting service.
+    // TODO: Continue scanning once drivelist is fixed (see below)
     driveScanner.stop();
 
-    return exceptionReporter.report(error);
+    // Clear the drive list, to avoid displaying stale data
```

Comment:
Just a measure against accidental flashing of the wrong device until https://github.com/resin-io-modules/drivelist/pull/202 is merged; because if devices are unplugged or new ones plugged in the device numbers could mismatch what's displayed in case of an error. And to avoid Etcher nuking devices it shouldn't, the list is cleared.